Podcast/BookmarkViewController.swift

```diff
@@ -52,6 +56,11 @@ class BookmarkViewController: ViewController, UITableViewDelegate, UITableViewDa
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
+        if self.episodes.isEmpty {
```

Comment:
We should figure out a way to integrate this into a tableview subclass so we don't need this if-statement over and over again.

Reply:
yea agreed I can do this .. i'll implement it on the views that have an empty state for now!